python/opae.admin/opae/admin/tools/opaevfio.py

```diff
@@ -183,14 +183,28 @@ def initialize_vfio(addr, new_owner, enable_sriov):
         load_driver('vfio-pci')
 
     print('Binding {} to vfio-pci'.format(msg))
-    new_id = '/sys/bus/pci/drivers/vfio-pci/new_id'
-    try:
-        with open(new_id, 'w') as outf:
-            outf.write('{} {}'.format(vid_did[0], vid_did[1]))
-    except OSError as exc:
-        if exc.errno != errno.EEXIST:
-            print(exc)
+
+    # On Linux kernel >= 3.15, use driver_override to bind the
+    # vfio-pci driver, which ensures the driver binds only to
+    # the given function. On older kernels, the driver will
+    # instead bind to all functions of the associated device.
```

Comment:
Same comment as above.  The behavior seems to be coupled to the use of new_id and not necessarily older kernels.  Could you clarify the comments?